cnf/ext/repositories.bnd

```diff
@@ -48,15 +48,15 @@ mavencentral:           https://repo.maven.apache.org/maven2
         readOnly=true
 
 # m2e debug repos
-#-plugin.m2e.debug:\
-#    aQute.bnd.repository.p2.provider.P2Repository; \
-#        name="Eclipse EMF 2.13"; \
-#        url="https://download.eclipse.org/modeling/emf/emf/builds/release/2.13";\
-#        location="${workspace}/cnf/cache/stable/EclipseEMF",\
-#    aQute.bnd.repository.p2.provider.P2Repository; \
-#        name="Eclipse Webtools 3.6.3"; \
-#        url="https://download.eclipse.org/webtools/downloads/drops/R3.6.3/R-3.6.3-20150216091848/repository/";\
-#        location="${workspace}/cnf/cache/stable/EclipseWTP",\
+-plugin.m2e.debug:\
+    aQute.bnd.repository.p2.provider.P2Repository; \
+        name="Eclipse EMF 2.13"; \
+        url="https://download.eclipse.org/modeling/emf/emf/builds/release/2.13";\
```

Comment:
This is a no-no. Relying on Eclipse download servers kills the reliability of our builds which is why we no longer do. We now use https://dl.bintray.com/bndtools/eclipse-repo/4.7.3a/. 

This section is commented out for normal builds and only to be uncommented for debugging purposed. If you want to add the desired jars to the normal build, they need to be added to https://dl.bintray.com/bndtools/eclipse-repo/4.7.3a/.

Reply:
> Relying on Eclipse download servers kills the reliability of our builds which is why we no longer do.
I have no problem with avoiding Eclipse download servers.

> This section is commented out for normal builds and only to be uncommented for debugging purposes.

For normal Bndtools development it's really necessary to be able to run up what you've just been working on without a full build cycle. This whole section was broken (i.e. it didn't work at all) because the previous download site had been archived. Even switching to the new archive location didn't work because the versions were too old for Bndtools new Eclipse baseline. My view is therefore that we need to maintain this piece of the build because it's important.

Reply:
OK, then someone will need to identify the necessary jars to add to the bintray repo so we can up date it.